### ISSUE TITLE
Write unit tests for `org.trebol.operation.services`

### ISSUE THREAD

**Issue by bglamadrid** [trebol-ecommerce/spring-boot-backend#88]
As the subject says, it is important to have the basics covered for all service classes within the package

`org.trebol.operation.services`

This issue may be blocked by future refactorings, nonetheless it will remain here as long as it doesn't have at least 60% code coverage.

**Comment by bglamadrid** [comment]
> @bglamadrid Can we split this up? Can I make separate issues for each class/test?

Yes, you can take one of these services and work on it. Does any one pick your interest right now?

**Comment by NyorJa** [comment]
Im gonna take also @bglamadrid

**Comment by bglamadrid** [comment]
> Im gonna take also @bglamadrid

Sure, @mepox just created the proper issues (thank you again!) and I wrote a little description for each of them with the intention.

@mepox also please pick, and request me to assign you, one of those.
